content/03.results.md

```diff
@@ -224,46 +235,47 @@ The mutual exclusivity of signatures 3 and MMR corroborates a previous report su
-Thus, we suspect that quanTIseq might actually be capturing microglia within these immune cell fractions.
-While we did not detect notable prognostic effects of immune cell infiltration in HGG or DMG, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 5.2e17, CI = 1.2e5-2.3e30, p = 0.006) in medulloblastoma tumors (**Figure {@fig:Fig5}D**).
-Medulloblastomas typically have low expression of _CD274_ (PD-L1) [@doi:10.18632/oncotarget.24951] a result we reproduced here (**Figure {@fig:Fig5}E**).
-However, we also found that higher expression of _CD274_ is significantly associated with better overall prognosis (HR = 0.00074, CI = 3.0e-6 to 0.18, p = 0.01).
-Consistent with a previous study, we found higher expression of _CD274_ in the WNT subgroup, which has the best prognosis of all medulloblastoma subgroups [@doi:10.1080/2162402X.2018.1462430] (**Figure {@fig:Fig5}E**).
-Finally, we asked whether any molecular subtypes might show an immunologically hot phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells and while we do see the highest ratios in Group 3 and Group 4 medulloblastomas (**Figure {@fig:S6}F**), very few tumors had ratios > 1, highlighting an urgent need to identify novel therapeutics for these immunologically cold pediatric brain tumors with poor prognosis.
-
-![**Transcriptomic and immune landscape of pediatric brain tumors** A, First two dimensions from UMAP of sample transcriptome data. Points are colored by `broad_histology` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, Box plots of quanTIseq estimates of immune cell proportions in histologies with N > 15. D, Forest plot depicting the effect of immune cell proportion or _CD274_ expression on overall survival of medulloblastomas. Hazard ratio (HR) with 95% confidence intervals and p-values are listed and significant p-values are denoted with colored diamonds. Of note, the Macrophage M1 HR was 0 (coefficient = -8.95e04) with infinite upper and lower CIs, thus it was not plotted. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Adjusted p-values are shown.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure5.png?sanitize=true){#fig:Fig5 width="7in"}
\ No newline at end of file
+Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in both embryonal and diffuse astrocytic and oligodendroglial tumors compared to several other cancer groups.
```

Comment:
Is "Bonferroni-corrected" is redundant, ie, "corrected" is understood?

Reply:
_No, there are actually _many_ different ways to correct p-values! Just saying "corrected" along does not imply Bonferroni._

Reply:
oh, no I meant, saying "Bonferroni" == it has been corrected :)